processing/src/main/java/io/druid/query/filter/DruidDoublePredicate.java

```diff
@@ -19,12 +19,32 @@
 
 package io.druid.query.filter;
 
-
+@SuppressWarnings("unused")
```

Comment:
Please add a comment explaining why you had to add this annotation.

Reply:
I added this as all implementations of DruidDoublePredicate use lambda and intellij was wrongly complaining that the parameter input is unused.